cmd/initialize_with_genesis.cpp

```diff
@@ -0,0 +1,258 @@
+
+static void check_rlp_err(rlp::DecodingResult err) {
+    if (err != rlp::DecodingResult::kOk) {
+        throw err;
+    }
+}
+
+static intx::uint256 convert_string_to_uint256(std::string const& str) {
+    intx::uint256 res{0};
+    for(size_t i = 0; i < str.size(); i++) {
+        auto current_digit{str.at(i) - '0'};
+        res += current_digit;
+        res *= 10;
+    }
+    return res/10;
+}
+
+static bool is_hex(std::string const& s) {
+  return s.compare(0, 2, "0x") == 0
+      && s.size() > 2
+      && s.find_first_not_of("0123456789abcdefABCDEF", 2) == std::string::npos;
+}
+
+static uint64_t get_uint_from_field(std::string const& field) {
+    int form = is_hex(field)? 16 : 10;
+    uint64_t res = strtoull(field.c_str(), nullptr, form);
```

Comment:
No need to check if isHex.
Providing 0 to the radix (third parameter) strtoull automatically detects the form as :
* if string begins with a digit not equal to `0` then is decimal
* if string begins with a digit equal to `0` then is octal
* if string begins with `0x` then is hexadecimal

By consequence I believe the above is_hex is pleonastic